From d905f54136575ccf815b676295b3b0a9b6b8b979 Mon Sep 17 00:00:00 2001
From: Robin Murphy <robin.murphy@arm.com>
Date: Mon, 26 Jan 2026 12:32:03 +0000
Subject: [PATCH 138/187] FROMLIST(v1): arm64: dts: rockchip: Clean up
 NanoPi-R2S Plus

Apparently something went wonky in the refactoring, and NanoPi R2S Plus
has ended up "overriding" the GMAC properties from the base R2S include
with all identical values. Clean up the redundancy.

Signed-off-by: Robin Murphy <robin.murphy@arm.com>
---
 .../dts/rockchip/rk3328-nanopi-r2s-plus.dts     | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s-plus.dts b/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s-plus.dts
index f72b1518c14f..4eff0503928a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s-plus.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3328-nanopi-r2s-plus.dts
@@ -29,20 +29,3 @@ &emmc {
 	pinctrl-0 = <&emmc_clk &emmc_cmd &emmc_bus8>;
 	status = "okay";
 };
-
-&gmac2io {
-	phy-handle = <&rtl8211e>;
-	tx_delay = <0x24>;
-	rx_delay = <0x18>;
-
-	mdio {
-		rtl8211e: ethernet-phy@1 {
-			reg = <1>;
-			pinctrl-0 = <&eth_phy_reset_pin>;
-			pinctrl-names = "default";
-			reset-assert-us = <10000>;
-			reset-deassert-us = <50000>;
-			reset-gpios = <&gpio1 RK_PC2 GPIO_ACTIVE_LOW>;
-		};
-	};
-};
-- 
2.43.0

